pkg/apis/core/types.go

```diff
@@ -2160,6 +2160,25 @@ type ResourceRequirements struct {
 	// otherwise to an implementation-defined value
 	// +optional
 	Requests ResourceList
+	// Claims lists the names of resources, defined in spec.resourceClaims,
+	// that are used by this container.
+	//
+	// This is an alpha field and requires enabling the
+	// DynamicResourceAllocation feature gate.
+	//
+	// This field is immutable.
+	//
+	// +featureGate=DynamicResourceAllocation
+	Claims []ResourceClaim
```

Comment:
An empty list is valid, so yes.

Do I need to add `+optional` to document that? As discussed elsewhere with Tim, the tag does not affect code generation. It would be only for users.

Reply:
As mentioned elsewhere I added `+optional` to be on the safe side.